crates/astria-conductor/src/executor/mod.rs

```diff
@@ -305,6 +249,18 @@ impl Executor {
         ExecutorBuilder::new()
     }
 
+    pub(super) fn celestia_channel(&self) -> mpsc::UnboundedSender<Vec<SequencerBlockSubset>> {
+        self.celestia_tx.upgrade().expect(
+            "should work because the channel is held by self, is open, and other senders exist",
+        )
+    }
```

Comment:
can this be called multiple times to return multiple senders?

Reply:
Yes. Although I am going to change the `celestia_tx -> UnboundedSender` and the `upgrade -> clone`

Reply:
Does no longer apply because we don't use weak channels (was buggy; see PR #711)